pkg/util/timeutil/time_zone_util.go

```diff
@@ -94,8 +95,9 @@ func TimeZoneStringToLocation(
 //
 // The strings produced by FixedOffsetTimeZoneToLocation look like
 // "<fixedOffsetPrefix><offset> (<origRepr>)".
-// TODO(#42404): this is not the format given by the results in
-// pgwire/testdata/connection_params.
+// The old <origRepr> is simply a number.
+// The new <origRepr> is formatted <-%s>+%s or <+%s>+%s.
+// We have to be able to format both versions.
 func ParseFixedOffsetTimeZone(location string) (offset int, origRepr string, success bool) {
```

Comment:
can we remove the "fixed offset" timezone name, and name all timezones this way?
i.e. currently time zone locations right now are named "fixed offset: 1.23 (-1.23)". let's just follow postgres' style instead, and for SHOW TIME ZONE we just have to output that location name.

Reply:
So I changed it such that for offsets where it's a numeric, it uses the `<+-%d>%d` formatting but I left the fixed offset since its still used for something like `GMT +3`

Reply:
is that the same as pg?

looks like they use the GMT+3 name instead in that case:

```

otan=# set time zone 'GMT+3';
SET
otan=# show time zone;
 TimeZone
----------
 GMT+3
(1 row)
```

do we do the same?

Reply:
seems like we do right now. is the `fixed offset:` prefix still needed then?

Reply:
yeah we do the same, demo@127.0.0.1:26257/movr> set time zone 'UTC+3';
warning: cannot save command-line history: write_history: illegal byte sequence
SET


Time: 1ms total (execution 1ms / network 0ms)

demo@127.0.0.1:26257/movr> show time zone;
  timezone
------------
  UTC+3
  
I left this `fixed offset: prefix` to handle this case, not sure if its still needed.

Reply:
i think we should delete it - it's no longer used.

Reply:
Removed.